pyrtools/tools/display.py

```diff
@@ -461,6 +497,16 @@ def imshow(image, vrange='indep1', zoom=1, title='', col_wrap=None, ax=None,
     image_tmp = []
     title_tmp = []
     for img, t in zip(image, title):
+        if as_rgb and (img.ndim == 2 or img.shape[-1] not in [3, 4]):
```

Comment:
I'm wondering if `as_rgb = (img.shape[-1] in [3, 4])` is 100% correct? If so, it is possible to reduce another parameter.

Reply:
sorry `as_rgb = (img.ndim == 3) and (img.shape[-1] in [3, 4])`, and assuming there is no user mistake.

Reply:
That's true, we can probably get rid of that. I think it would be `as_rgb = (img.shape[-1] in [3,4] and img.ndim==3)` (because if someone passes an image with shape `(4,4)`, that should be a grayscale image). 

Currently, we don't allow mixing of RGB and grayscale images, but if we do something like the above to automatically infer whether each image should be RGB or not, then we should be able to.

Reply:
Alright, made those changes